### PR TITLE
docs: add basic docs for bulk search API

### PR DESCRIPTION
**Pull request by eddieantonio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,42 +339,228 @@ some entries due to the nature of the tests.
...
+ - `not_found` is an array of strings; each string is a wordform that
+   has no recordings.
+
+
+#### Errors
+
+speech-db will not explicitly issue errors, however application servers
+and proxies may drop or reject large requests. If speech-db is running
+in front of an application server (e.g., `uwsgi`), you may get **502 Bad
+Gateway** errors when too many wordforms are request. The fix is to
```
Comment (nienna73) [inline README.md]:
```suggestion
Gateway** errors when too many wordforms are requested. The fix is to
```

File: README.md [hunk 2]
```diff
@@ -339,42 +339,228 @@ some entries due to the nature of the tests.
...
+
+
+### Legacy recording search
+
+URL:
+
+    /recording/_search/{query}
+
+Where `{query}` is replaced by one ore more Cree word forms, written in
+SRO. If more than one word form is supplied, each word form must be
```
Comment (nienna73) [inline README.md]:
```suggestion
SRO. If more than one wordform is supplied, each wordform must be
```
(there's one more on the line above)

**Review by nienna73 — APPROVED** [review]
LGTM just a couple typos

File: README.md [hunk 3]
```diff
@@ -339,42 +339,228 @@ some entries due to the nature of the tests.
...
+```
+
+
+### Legacy recording search
+
+URL:
+
+    /recording/_search/{query}
+
+Where `{query}` is replaced by one ore more Cree word forms, written in
```
Comment (eddieantonio) [inline README.md]:
```suggestion
Where `{query}` is replaced by one ore more Cree wordforms, written in
```

**Pull request merged**